x/btcstaking/types/ibc_callbacks.go

```diff
@@ -0,0 +1,29 @@
+package types
+
+const (
+	// CallbackActionAddBsnRewardsMemo is the memo string indicating BSN reward distribution
+	CallbackActionAddBsnRewardsMemo = "add_bsn_rewards"
+)
+
+// CallbackMemo defines the structure for callback memo in IBC transfers
+type CallbackMemo struct {
+	Action string `json:"action,omitempty"`
```

Comment:
`Action, DestCallback, CallbackInfo.Address` are mandatory fields checked by the ICS20 middleware wrapper to call the callback interface